src/components/AddUser.tsx

```diff
@@ -16,13 +48,18 @@ const AddUser: FC = () => {
                 label="Name"
                 name="name"
                 autoFocus
+                onChange={handleInp}
+                value={user.name}
             />
             <TextField
                 margin="normal"
                 fullWidth
                 name="salary"
```

Comment:
The 'name' properties for 'input' fields (such as TextField component)  are used to grab the form values after the form submission. If you treat your 'inputs' as a pieces of a form, please consider using the form-managing library like 'react-hook-forms' (https://www.react-hook-form.com/)